### DISCUSSION TITLE
How to uninstall or reinstall nuclei?

### DISCUSSION THREAD

**Discussion by deadjdona** [projectdiscovery/nuclei#3309] [Q&A] [answered]
### Nuclei version:
Version 1 and latest version.

### Current Behavior:
Can not install or "uninstall" nuclei.

### Expected Behavior:
Can install nuclei.

### Steps To Reproduce:

accidentally run:
1. go install -v github.com/projectdiscovery/nuclei/cmd/nuclei@latest

then this do not work:
2. go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest

### Anything else:
Can not remove old version or install new version:

````└─$ go install -v github.com/projectdiscovery/nuclei/v2/cmd/nuclei@latest
github.com/projectdiscovery/utils/maps
github.com/projectdiscovery/clistats
github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/hosterrorscache
# github.com/projectdiscovery/utils/maps
go/pkg/mod/github.com/projectdiscovery/utils@v0.0.9/maps/synclock_map.go:15:18: undefined: atomic.Bool
note: module requires Go 1.19
# github.com/projectdiscovery/nuclei/v2/pkg/protocols/common/hosterrorscache
go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.8.9/pkg/protocols/common/hosterrorscache/hosterrorscache.go:36:16: undefined: atomic.Int32
go/pkg/mod/github.com/projectdiscovery/nuclei/v2@v2.8.9/pkg/protocols/common/hosterrorscache/hosterrorscache.go:116:40: undefined: atomic.Int32
note: module requires Go 1.19
# github.com/projectdiscovery/clistats
go/pkg/mod/github.com/projectdiscovery/clistats@v0.0.12/clistats.go:86:30: undefined: atomic.Uint64
go/pkg/mod/github.com/projectdiscovery/clistats@v0.0.12/clistats.go:117:37: undefined: atomic.Uint64
go/pkg/mod/github.com/projectdiscovery/clistats@v0.0.12/fields.go:10:23: undefined: atomic.Uint64
note: module requires Go 1.19

**Comment by ehsandeep** [comment, marked as answer]
@deadjdona as the error says - `note: module requires Go 1.19`

You need to have go 1.19 installed on your system to install nuclei from the source, you probably have outdated version of go installed that you can confirm with `go version`

**Comment by deadjdona** [comment]
thanks, $GOLANG nd $GOPATH were messed up, somehow I had 2 versions installed.
all solved now.
